x/language.rb

```diff
@@ -0,0 +1,18 @@
+module X
+  class Language
+    def self.of(id)
+      tracks[id.to_s]
+    end
+
+    def self.tracks
+      @@tracks ||= fetch_tracks.inject({}) do |hash, track|
```

Comment:
You're caching two values, `@@tracks` and `@@fetched_tracks`. It seems like we should only need to cache one of these values.